angr/exploration_techniques/loop_seer.py

```diff
@@ -56,36 +58,20 @@ def setup(self, simgr):
             l.warning("LoopSeer uses normalized CFG. Recomputing the CFG...")
             self.cfg.normalize()
 
-        if type(self.functions) is str:
-            func = [self.cfg.kb.functions.function(name=self.functions)]
-
-        elif type(self.functions) is int:
-            func = [self.cfg.kb.functions.function(addr=self.functions)]
-
-        elif type(self.functions) is Function:
-            func = [self.functions]
+        func = []
+        if type(self.functions) in (str, int, Function):
+            func.extend(self._get_function(self.functions))
```

Comment:
Nitpick: The same idea applies to `_get_function()`. Does it return a collection of functions? If so, it's better named `_get_functions()`.